common/djangoapps/entitlements/api/v1/tests/test_views.py

```diff
@@ -132,6 +134,47 @@ def test_staff_get_all_entitlements(self):
         results = response.data.get('results', [])
         assert results == CourseEntitlementSerializer(entitlements, many=True).data
 
+        # Make sure that these results aren't expired
+        assert results[0].get('expired_at') is None and results[1].get('expired_at') is None
```

Comment:
Is this meant to test that it doesn't run the expiration logic? If so, you should show that the entitlements would have otherwise been expired if we had checked, but weren't because of the special logic we added. For example, you could add an assertion verifying that `entitlements[0].expired_at_datetime` is `None`.

Reply:
nvm, I see that you test that explicitly in the next test.